packages/lesswrong/lib/collections/posts/custom_fields.ts

```diff
@@ -793,7 +793,7 @@ addFieldsDict(Posts, {
     viewableBy: ['guests'],
     insertableBy: ['members'],
     editableBy: [userOwns, 'sunshineRegiment', 'admins'],
-    hidden: (props) => !props.eventForm,
+    hidden: (props) => !props.eventForm || forumTypeSetting.get() === 'EAForum',
```

Comment:
You can probably put EAForum first here, and have it only be a function if !isEAForum.